### PR TITLE
Add SSL_DOMAIN environment variable for custom Common Name

### PR DESCRIPTION
**Pull request by michaelservidio**
Create environment variable `SSL_DOMAIN` (optionally set and pulled from value in `.env`) which will be used in `server.pem` creation. 

Allows self-signed certificates to have a custom Common Name (instead of localhost, which would still be the default).

Use case:
This allowed me to view my local application at https://mydomain.test by setting `SSL_DOMAIN=*.mydomain.test` in `.env`

### CODE REVIEW

File: stubs/nginx.stub [hunk 1]
```diff
@@ -7,11 +7,12 @@
...
         volumes:
             - 'sail-nginx:/etc/nginx/certs'
             - './vendor/ryoluo/sail-ssl/nginx/templates:/etc/nginx/templates'
             - './vendor/ryoluo/sail-ssl/nginx/generate-ssl-cert.sh:/docker-entrypoint.d/99-generate-ssl-cert.sh'
         depends_on:
             - ${APP_SERVICE:-laravel.test}
         networks:
-            - sail
\ No newline at end of file
+            - sail
```
Comment (ryoluo) [inline stubs/nginx.stub:18]: Do not insert final new line.

**Review by ryoluo — CHANGES_REQUESTED** [review]
Thank you for PR!
Could you update README to explain new env var?

File: stubs/nginx.stub (hunk 1 above)
Comment (michaelservidio) [reply stubs/nginx.stub]: Changed .editorconfig: `insert_final_newline = false`

**Comment by michaelservidio** [comment]
For Chrome to accept the certificate as valid, it required a subjectAlternativeName to be set. Added another variable, `SSL_ALT_NAME`, to specify (e.g. `SSL_ALT_NAME=DNS:localhost,DNS:mydomain.test`).

Updated `README.md` to explain both variables.

**Pull request merged**